### DISCUSSION TITLE
Discogs - Cart

### DISCUSSION THREAD

**Discussion by magnaux** [Purfview/Discogs-Scout#13] [Ideas]
possible support cart page too ? 

![Screenshot 2022-09-07 at 03-32-36 Shopping Cart](https://user-images.githubusercontent.com/107857394/188763557-c81397af-3f0a-4899-af9f-78dbabc4c9f2.png)
![Screenshot 2022-09-01 at 03-16-12 Shopping Cart](https://user-images.githubusercontent.com/107857394/188763564-6ffb7eca-cfc8-4990-9a49-ed271ebc3805.png)

thanks alot!

**Comment by magnaux** [comment]
also if there could be a Create Request that could fill in release details <3

and also if possible to Search Artist or album name or Label , when clicking on each

**Comment by Purfview** [comment]
> also if there could be a Create Request that could fill in release details <3

What this means?

> and also if possible to Search Artist or album name or Label , when clicking on each

I've no idea what this means too.

**Comment by magnaux** [comment]
> > also if there could be a Create Request that could fill in release details <3
> 
> What this means?

Create Request on RED,



> > and also if possible to Search Artist or album name or Label , when clicking on each
> 
> I've no idea what this means too.

Clicking on Artist or album or Label  on discogs to take me straight to RED search instead

**Comment by Purfview** [comment]
This script is for Discogs only, not for RED or any other site.

**Comment by Purfview** [comment]
You don't need "Cart" page support anymore?

**Comment by magnaux** [comment]
i need , if possible, thanks!!

**Comment by Purfview** [comment]
I don't have "these items got away" section, I added bunch of stuff to cart so maybe after month or two I'll start seeing it. :)

But maybe it's better to add support for "sell" url instead, that's were it leads if you press links on those albums, and those pages looks similar to the already supported pages?

**Comment by magnaux** [comment]
> 
> ![Screenshot 2022-09-07 at 03-32-36 Shopping Cart](https://user-images.githubusercontent.com/107857394/188763557-c81397af-3f0a-4899-af9f-78dbabc4c9f2.png)

you mean here ?

**Comment by magnaux** [comment]
Also have been coming across more : 
![Screenshot 2022-09-10 at 17-55-21 Browse Torrents Redacted](https://user-images.githubusercontent.com/107857394/189489010-f64f081c-d7d2-4e1b-83df-d77f384d0f49.png)
![Screenshot 2022-09-10 at 17-54-47 Otaku Records (2)](https://user-images.githubusercontent.com/107857394/189489014-b8f8cc58-23a7-4763-be1d-1cd2d8cc57f2.png)

**Comment by Purfview** [comment]
> you mean here ?

I meant your second screen in the first post.

> Also have been coming across more :

I've no idea what this means.

> Also is possible add option skip check if Rating = (option to select how many stars ) or Note = ......._..._ just to make it a little faster when checking 250 rels per page on wantlist view (i am using rating system to mark what exists / owned / non exist )

This is not related to this issue/request, open a new one.
Probably possible, dunno when I'll check it.

**Comment by magnaux** [comment]
On discogs release has EP on site it doesnt ... maybe just dont add "EP" on search ... or add option for words not to use 

photo 2 discogs (shows yellow)
photo 1 site (release with out "EP")

> > Also have been coming across more :
> 
> I've no idea what this means.
>

**Comment by magnaux** [comment]
hi, any progress ? :)

**Comment by Purfview** [comment]
It's not ready for me to looked at it, yet:

> I don't have "these items got away" section, I added bunch of stuff to cart so maybe after month or two I'll start seeing it.

**Comment by magnaux** [comment]
how about what is added in the cart before check out ?

**Comment by magnaux** [comment]
> > ![Screenshot 2022-09-07 at 03-32-36 Shopping Cart](https://user-images.githubusercontent.com/107857394/188763557-c81397af-3f0a-4899-af9f-78dbabc4c9f2.png)
>

**Comment by Purfview** [comment]
Didn't looked at it, waiting till I can see all stuff on that page.

**Comment by magnaux** [comment]
Cool, thanks!

**Comment by Purfview** [comment]
I don't see point in supporting this page, you buy release or not, if you don't want to buy yet then just add to wantlist.

"Looks like these items got away" has button "Move All To My Wantlist", so there is no point for it too.

**Reply by magnaux** [reply]
since there are many sellers .... selling the same vinyl in different condition and different postage , i am making multiple orders 
and its nice to see availability , my want list is 6500 items  so i cant remember everything :(
